src/backend/src/prisma/seed-data/users.seed.ts

```diff
@@ -700,6 +700,87 @@ const snowBite: Prisma.UserCreateInput = {
   role: Role.MEMBER
 };
 
+const zayFlowers: Prisma.UserCreateInput = {
+  firstName: 'Zay',
+  lastName: 'Flowers',
+  googleAuthId: '8172979',
+  email: 'flowers.za@husky.neu.edu',
```

Comment:
if you look at the other users in the ravens team, you'll see that their emails end in @ravens.com , match that for all the new team members you've made